vizro-core/src/vizro/models/_controls/filter.py

```diff
@@ -119,16 +119,20 @@ def _set_slider_values(self):
         if isinstance(self.selector, SELECTORS["numerical"]):
             if self._column_type != "numerical":
                 raise ValueError(
-                    f"Chosen selector {self.selector.type} is not compatible with column_type {self._column_type}."
+                    f"Chosen selector {self.selector.type} is not compatible "
+                    f"with {self._column_type} column '{self.column}'."
                 )
             min_values = []
             max_values = []
             for target_id in self.targets:
                 data_frame = data_manager._get_component_data(target_id)
                 min_values.append(data_frame[self.column].min())
                 max_values.append(data_frame[self.column].max())
-            if not is_numeric_dtype(min(min_values)) or not is_numeric_dtype(max(max_values)):
-                raise ValueError(f"No numeric value detected in chosen column {self.column} for numerical selector.")
+            if not is_numeric_dtype(pd.Series(min_values)) or not is_numeric_dtype(pd.Series(max_values)):
```

Comment:
and `is_numeric_dtype` is not deprecated? Curious!

Reply:
Doesn't seem like it, we only got a warning for is_period_dtype

<img width="987" alt="Screenshot 2023-09-26 at 10 04 16" src="https://github.com/mckinsey/vizro/assets/90609403/b61d7c0c-566e-4aed-a6ec-fa52f4584056">